services/web-app/utils/auth-checkers/validUserAuthorizationChecker.js

```diff
@@ -0,0 +1,22 @@
+import { retrieveUser } from '../retrieveUser'
+
+/*
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+/**
```

Comment:
Looks like this copyright header ended up in the wrong place. Should be above the imports. IDK exactly what causes this, but I've seen it once or twice myself.

Reply:
Done!